README.md

```diff
@@ -30,5 +30,322 @@
 
 # Triton Inference Server
 
-**NOTE: You are currently on the r22.03 branch which tracks stabilization
-towards the next release. This branch is not usable during stabilization.**
+Triton Inference Server provides a cloud and edge inferencing solution
+optimized for both CPUs and GPUs. Triton supports an HTTP/REST and
+GRPC protocol that allows remote clients to request inferencing for
+any model being managed by the server. For edge deployments, Triton is
+available as a shared library with a C API that allows the full
+functionality of Triton to be included directly in an
+application.
+
+## What's New in 2.20.0
+- Models can now [load from a serialized model_config message](docs/protocol/extension_model_repository.md#load) with the Triton Server API.
+- ONNX Runtime, TensorRT, and Tensorflow backends now support server-side, multi-dimensional [ragged batching](docs/ragged_batching.md).
+- [Cache miss statistics](docs/metrics.md) have been added to the Prometheus metrics.
+- Trace settings can be configured with the [Triton Server Trace Protocol](docs/protocol/extension_trace.md).
+- Refer to the 22.03 column of the [Frameworks Support Matrix](https://docs.nvidia.com/deeplearning/frameworks/support-matrix/index.html) for container image versions on which the 22.03 inference server container is based.
```

Comment:
This line does not get added to the What's New section. Remove

Reply:
Done